server/models/index.js

```diff
@@ -1 +1,76 @@
-index.js
\ No newline at end of file
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+mongoose.connect('mongodb://localhost/hunch');
+
+
+const UserSchema = new Schema({
+    name: {
+        type: String
+    },
+    points: {
+        type: Number
+    },
+    answer: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Answer'
+    }
+})
+
+const TopicSchema = new Schema({
+    topic: {
+        type: String
+    }
+})
+
+const GameSchema = new Schema({
+    currentTopic: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Topic' },
+    round: {
+        type: Number
+    },
+    users: [UserSchema],
+    topics: [TopicSchema]
```

Comment:
similar thing here, except I'm pretty sure that topics should be an independent collection and this should be an array of references. otherwise you're going to duplicate data from the topics collection here.
`topics: [{type: Schema.Types.ObjectId, ref: 'Topic'}]`